pom.xml

```diff
@@ -96,6 +98,25 @@
             <artifactId>spring-webmvc</artifactId>
             <version>${spring.version}</version>
         </dependency>
+
+        <dependency>
+            <groupId>com.atlassian.connect</groupId>
+            <artifactId>atlassian-connect-spring-boot-starter</artifactId>
+            <version>1.4.2</version>
+        </dependency>
```

Comment:
is the maven swagger in this branch?

Reply:
For some reason it's not, don't merge this one I will create another pull request